src/components/map/thumbnailMap.js

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import { Row } from "react-bootstrap";
+import { GeoJSON, MapContainer } from "react-leaflet";
+import bbox from "@turf/bbox";
+
+import useDataMaps from "../../hooks/useDataMaps";
+import {
+  tierColors,
+} from "../../utils/utils";
+
+import * as styles from "./map.module.css";
+import useGeoJson from "../../hooks/useGeoJson";
+
+export default function ThumnailMap({
+  dimensions,
+  dataId, // state or OPO abbr (e.g. AZ or ALOB)
+  view, // "state" or "opo"
+}) {
+  const [{ opoDataMap, stateDataMap }] = useDataMaps();
+  const { dsaGeoData, statesGeoData } = useGeoJson();
+
+  // compose fill OPO geoJson with appropriately filtered
+  // OPO features based on view ("opo" or "state")
+  const fillGeoJson =
+  {
+    ...dsaGeoData?.childGeoJson,
+    features: getOpoFeatures(view, dsaGeoData.childGeoJson.features, opoDataMap, dataId)
+      .map(f => ({
+        ...f,
+        properties: {
+          ...f.properties,
+          tier: opoDataMap[f.properties.abbreviation].tier,
+        }
+      }))
+  };
+
+  // Compose boundary state geoJson with appropriately filtered
+  // state features based on view ("opo" or "state")
+  const boundaryGeoJson = {
+    ...statesGeoData.childGeoJson,
+    features: getStateFeatures(view, statesGeoData.childGeoJson.features, dataId)
+      .map(f => ({
+        ...f,
+        properties: {
+          ...f.properties,
+          name: stateDataMap[f.properties.abbreviation].name,
+        },
+      })),
+  };
+
+  // Generate bounding box
+  // - For state view, use boundary geoms
+  //  (which will only include single state given by dataId)
+  // - For OPO view, use fill geoms
+  //  (which will only include single OPO given by dataId)
+  const [minX, minY, maxX, maxY] = bbox(
+    view === "state"
+      ? boundaryGeoJson
+      : fillGeoJson
+  );
+
+  return (
+    <Row className={styles.map}>
+      <div style={dimensions}>
+        {
+          // Hack: [`window` dependency for Leaflet](https://www.gatsbyjs.com/docs/debugging-html-builds/#fixing-third-party-modules)
+          typeof window !== "undefined" && (
+            <MapContainer
+              key={dataId.name + " container"}
+              bounds={[
+                [minY, minX],
+                [maxY, maxX],
+              ]}
+              scrollWheelZoom={false}
+              style={{...dimensions,  backgroundColor: "#fff" }}
+              zoomControl={false}
+              touchZoom={false}
+              dragging={false}
+              className={styles.mapContainer + " thumbnail"}
+            >               
+              {/* Create layer for all state polygons with fill (to contextualize other geoms!) */}
+              <GeoJSON
+                key={dataId + "state-fill"}
+                data={{...statesGeoData.childGeoJson}}
+                interactive={false}
+                style={{
+                  fillColor: "blue",

```

Comment:
update state fill color here

Reply:
do we know what color we want this updated to yet?